perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/build.sh

```diff
@@ -83,7 +82,8 @@ fi
 # Executing perf tests
 LOG_FILE_FIO_TESTS="${KOKORO_ARTIFACTS_DIR}/gcsfuse-logs${EXPERIMENT_NUMBER}.txt"
 GCSFUSE_FIO_FLAGS="$GCSFUSE_FLAGS --log-file $LOG_FILE_FIO_TESTS --log-format \"text\" --stackdriver-export-interval=30s"
-./run_load_test_and_fetch_metrics.sh "$GCSFUSE_FIO_FLAGS" "$UPLOAD_FLAGS"
+BUCKET_NAME="periodic-perf-tests${EXPERIMENT_NUMBER}"
```

Comment:
nit: can we name it experimental-periodic-perf-tests-${EXPERIMENT_NUMBER}?

Also, why are we keeping creds.json in the bucket?

Reply:
fixed
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/run_load_test_and_fetch_metrics.sh#L53

this script is common for both the builds and it is downloading creds.json file to upload data on gsheet.

Reply:
As discussed offline, anyway we are not gonna upload to gsheet as part of periodic_experiments. So, we will not keep creds.json in the experimental-periodic-perf-tests-${EXPERIMENT_NUMBER} bucket.